docs/tre-workspace-authors/firewall-rules.md

```diff
@@ -1,128 +1,256 @@
+This guide explains how to configure firewall rules for your Azure TRE environment.
 
-A TRE service may require certain firewall rules to be opened in the TRE firewall. Examples include:
+## Contents
+- [Overview](#overview)
+- [Types of Firewall Rules](#types-of-firewall-rules)
+- [Configuring Application Rules](#configuring-application-rules)
+- [Configuring Network Rules](#configuring-network-rules)
+- [Applying Your Configuration](#applying-your-configuration)
+- [Examples](#examples)
 
-- Access to an external authorisation endpoint
-- Access to an external data store
-- Access to an external API
+## Overview
 
-Please be aware when opening firewall rules there is the potential for data to be leaked from the workspace to the external location.
+The Azure TRE Firewall shared service controls network traffic flowing in and out of your research environment. The firewall configuration is defined using a JSON file that specifies rule collections and their associated rules.
 
-## Using Terraform to open firewall rules
+## Types of Firewall Rules
 
-Until a mechanism to update shared services has been implemented, firewall rule updates should be done using terraform as part of the service deployment. The aim is to create a firewall rule that grants access from the workspace's address space to the external location. A challenge with this is that the rule must use a priority that has not been used by any other rule.
+The Azure TRE Firewall supports two types of rules:
 
-1. Create a `firewall.tf` file in the `terraform` directory of the workspace.
+1. **Application Rules**: Control outbound access to specific websites/FQDNs
+2. **Network Rules**: Control traffic based on source, destination, port, and protocol
 
-1. Add the following code to the `firewall.tf` file to enable the TRE firewall and workspace network to be referenced:
+## Configuring Application Rules
 
-    ```terraform
-    data "azurerm_firewall" "fw" {
-        name                = "fw-${var.tre_id}"
-        resource_group_name = "rg-${var.tre_id}"
-    }
-
-    data "azurerm_virtual_network" "ws" {
-        name                = "vnet-${var.tre_id}-ws-${var.workspace_id}"
-        resource_group_name = data.azurerm_resource_group.ws.name
-    }
-    ```
+Application rules control HTTP/HTTPS traffic to specific web destinations.
 
-1. Define a local variable that contains the locations that access should be allowed to, and the naming format for the service resources for example:
+To configure application rules:
 
-    ```terraform
-    locals {
-        allowed_urls                     = ["*.anaconda.com", "*.anaconda.org"]
-        service_resource_name_suffix    = "${var.tre_id}-ws-${var.workspace_id}-svc-${local.service_id}"
-    }
-    ```
-
-1. Log into the Azure CLI using service principal details:
+1. Create a JSON file containing a `rule_collections` array
```

Comment:
Not a case of creating any json file? Need to reference where in 1tempalte_schema.json`